### DISCUSSION TITLE
This utility should be called winsudo

### DISCUSSION THREAD

**Discussion by adrian7** [microsoft/sudo#65] [Polls]
”Borrowing” the name or the traditional sudo command just adds a lot of confusion, for developers and users alike . 

I am an administrator of a forum for programmers, and we get our fair share of questions regarding installing software on WSL, which is an entire different operating system inside the Windows operating system. But thanks to good guys at Microsoft we don't have to care about the virtualization layer.
Of course the traditional sudo command comes up in every one of these discussions: `sudo apt-get ...` , `sudo useradd ...` and so on.

But now having a sudo command, yes _exactly_ the same name, on windows always begs an additional question when trying to help these people: "Are you using sudo for Windows", and the the newbie WSL user thinks this is a very dumb question - "_Of course I'm using sudo and I am on a Windows System!_", and they would be correct. 
However you have now to explain to them that there is a sudo command that you can run from the command prompt on Windows! and that the guy asking was just trying to get more details on their context. 

So not so sure about the name, maybe add a prefix to make it clear this is a windows specific utility?

**Comment by whosFritz** [comment]
Nah.... winduu it is

**Comment by AvogatoWizardWhisker** [comment]
I like the idea of `winsudo` so it won't create confusion between Linux's sudo. It will be also better if they add ab alias like `wsu` which is a short of Windows Super User. Or `wad` as Windows Admin do, since using it will elevate UAC and you're running the terminal as administrator.

**Comment by beedell-roke** [comment]
@adrian7, I disagree, because PowerShell Core allows my scripts to run cross-platform, where invoking the same `sudo`-named binary across all OSes matters. It'll be even better when https://github.com/PowerShell/PowerShell/issues/3232#issue-210886080 is implemented.

**Comment by AdreKiseque** [comment]
Aren't there several commands already that are the same between Windows and Linux? I mean PowerShell alone has a ton of bash commands as aliases for cmdlets. I don't see how sudo adds any to that confusion—if you need to ask someone if they're using Windows Sudo or Linux Sudo, just ask if they're using it inside WSL or in a Windows shell.

**Comment by gamingdivy** [comment]
I disagree, but if you want winsudo on your pc, just run this as administrator in cmd (only cmd, not powershell):
`mklink "%SYSTEMROOT%\winsudo.exe" "%SYSTEMROOT%\System32\sudo.exe"`
You can put it anywhere as long as it's in your environment variables path, paths is how windows instantly opens cmd or powershell when u type it in run box
if you want to put it somewhere like  your user instead of C:\Windows, open sysdm.cpl, goto advanced and click environment variables, then in "user variables" (or for global, select system variables) select path and click edit, you will see a list of variables, for your user theres only one, for system, theres multiple, in that tab, click "new", and type the path of where you typed winsudo
example you have winsudo in C:\winsudo\winsudo.exe, you make the path "C:\winsudo", you click ok and then ok again, now you can use winsudo
